oracle/oracle/rewards/controller.py

```diff
@@ -43,6 +43,22 @@
 w3 = Web3()
 
 
+class WithdrawalsCache:
+    block: BlockNumber
+    withdrawals: Wei
```

Comment:
Linters understand attributes from constructor, annotation here is not necessary